NLibrary/Classes/DashboardHomeMVVM/DashboardViewModel.swift

```diff
@@ -4,6 +4,7 @@ import AVKit
 
 @available(iOS 13.0, *)
 public class DashboardViewModel: DashboardViewModelProtocol {
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Lines should not have trailing whitespace.](https://app.codacy.com/manual/NathanDVT/NLibrary/pullRequest?prid=5376515)